css/style.css

```diff
@@ -1,4 +1,572 @@
+@charset "UTF-8";
+
+/*Обнуление*/
+
 @import url(../fonts/font.css);
+
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
+
+*, 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/chendler_tesla/pullRequest?prid=5700328)